internal/pkg/plan/persist.go

```diff
@@ -79,42 +113,21 @@ func (b *persistPlanBuilder) tryAddConfig(projectPath string, branch *model.Bran
 	}
 
 	// Create action
-	configKey := model.ConfigKey{BranchId: branch.Id, ComponentId: componentId}
-	actions := make([]PersistAction, 0)
-	action := &NewConfigAction{Key: configKey, PathInProject: path}
-	actions = append(actions, action)
-
-	// Search for config rows
-	for _, path := range b.UntrackedDirs() {
```

Comment:
Pred zmenou:
- ked sa nasiel novy `config`, tak sa nasledne spustil dalsi `for` a hladali sa `rows`.